tests/integration/conftest.py

```diff
@@ -355,10 +326,10 @@ def configured_project_with_label(client, rand_gen, annotation_submit_fn,
             "width": 50
         }
     }]
-    upload_task = MALPredictionImport.create_from_objects(
-        client, project.uid, f'mal-import-{uuid.uuid4()}', predictions)
+    upload_task = LabelImport.create_from_objects(
+        client, project.uid, f'label-import-{uuid.uuid4()}', predictions)
     upload_task.wait_until_done()
-    labels = annotation_submit_fn(project.uid, data_row.uid)
+    labels = [label.uid for label in project.labels()][0]
```

Comment:
why only the zeroth index?

Reply:
configured_project_with_label used to only return 1 id in the `labels` var, so the test that used it did this:

model_run.upsert_labels([label_id])

i can change the test itself to just take in (label_id) instead of ([label_id]) if that makes more sense, and to keep labels as the whole list

Reply:
actually looking at how the asserts are set up, it looks like it does expect it to just be one. let me know if i should change the asserts to account for multiple labels too, or if leaving it as the 0th index is okay

Reply:
ohh I was mislead by the variable name. labels is just a single label. Ok then you can just do:
```
label = next(project.labels()).uid
yield [project, label]
```
That way it won't confuse people in the future